### PR TITLE
Adds a note why `pidfile` integration test doesn't work

### PR DESCRIPTION
**Pull request by yihuaf**

### CODE REVIEW

**Comment by yihuaf** [comment]
This currently fails with:
```
not ok 1 - create with '--pid-file' option works
  ---
  {
    "error": "exit status 1",
    "stderr": "Error: bd521074-7261-452d-9a63-a75777fcc411 could not be killed because it was Stopped\n"
  }
  ...
1..1
```
and I can't reproduce this on my local box. Likely this will require tmate into the CI VM and see why the container process is failing.

**Comment by YJDoc2** [comment]
I'm not sure, but bors containers might help running github CI locally : https://hub.docker.com/r/borsng/dev and https://github.com/bors-ng/bors-ng
It states in the README setting up locally section :
>If you're using a macOS or Linux command line with Docker on it, ./script/setup && ./script/server will set up a local instance, with a mocked-out GitHub instance, using Docker to pull in all the underlying dependencies. 

So maybe this might help mock-up the github CI and check issue locally. Hope this helps :)

**Comment by yihuaf** [comment]
Nice. I will check it out. I was planning to put a tmate session into github action, so I can ssh into the actual machine:
```
https://github.com/mxschmitt/action-tmate
```
```
name: CI
on: [push]
jobs:
  build:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v2
    - name: Setup tmate session
      uses: mxschmitt/action-tmate@v3

```

**Comment by YJDoc2** [comment]
Oh, that is cool, I didn't know about that... thanks!

**Comment by yihuaf** [comment]
I couldn't identify the reason why it fails on the CI. Apparently `runc` also fails on the Github Action. So it is probably not worth it at this point to chase further. The test passes correctly on local box. I manually verified that the pid file created match the pid in the DEBUG log.

**Comment by utam0k** [comment]
@yihuaf Can I ask you to update the README.md?

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ youki is not at the practical stage yet. However, it is getting closer to practi
 |    Seccomp     |             Filtering system calls              |                                                  ✅                                                  |
 |     Hooks      | Add custom processing during container creation |                                                  ✅                                                  |
 |    Rootless    |   Running a container without root privileges   | It works, but cgroups isn't supported. WIP on [#77](https://github.com/containers/youki/issues/77)  |
-| OCI Compliance |        Compliance with OCI Runtime Spec         |                                   50 out of 53 test cases passing                                   |
+| OCI Compliance |        Compliance with OCI Runtime Spec         |                                   50 out of 52 test cases passing                                   |
```
Comment (yihuaf) [inline README.md:51]: Just to clarify, we reduce the number here when test also fails with `runc` right?

Comment (utam0k) [reply README.md:51]: :+1:

**Review by utam0k — APPROVED** [review]

**Pull request merged**